### PR TITLE
Players create light clouds

### PR DESCRIPTION
**Pull request by w1158mxs**
@erikbuck 
Is this good in the good?
issue#33 
see  block 66
"""!
Issue#33 and issue#34 were created on June 19 to let players create clouds anywhere and may build their homes on clouds
issue#42 was created on June 21 that block the user to build on clouds!
@w1158mxs left a preview on June 22 in issue#42 that will use the key it was mentioned in issue#33 and issue#34 to build clouds and will let players build their homes on clouds!
while issue#80 used keys
@w1158mxs uses another key to build clouds
@see README.md to know how to play it
"""

### CODE REVIEW

**Review by xieWright — COMMENTED** [review]
Looking at it, the changes I see work. It's pretty much just changing the property of the clouds so that you can destroy and build on them. However before merging this one, we will need to merge the dark clouds first since you made that a branch of this one.

**Review by bakursait2 — CHANGES_REQUESTED** [review]

**Comment by bakursait2** [comment]
> @erikbuck Is this good in the good? issue#33 see block 66 """! Issue#33 and issue#34 were created on June 19 to let players create clouds anywhere and may build their homes on clouds issue#42 was created on June 21 that block the user to build on clouds! @w1158mxs left a preview on June 22 in issue#42 that will use the key it was mentioned in issue#33 and issue#34 to build clouds and will let players build their homes on clouds! while issue#80 used keys @w1158mxs uses another key to build clouds @see README.md to know how to play it """

Sorry, I know it's directed to the Professor but I'd like to put forth my explanation and suggestion. 
You're absolutely right about the order of the issues (#33 and #34) being created before issue #42. The reason we implemented issue #42 first and then followed by issue #74 was to ensure a logical progression in creating clouds. In issue #1, we aimed to add the feature of moving clouds, which was one of the initial discussions. However, while issue #1 itself wouldn't directly conflict with issues #33 and #34, the adjustments made in this PR #105 and the next one #107 allow the user to build any block in the clouds, which wasn't mentioned as a requirement in both issues #33 and #34. This could create issues if we implemented the moving clouds feature, as moving clouds might leave other blocks in the sky without any clouds underneath them. So, to avoid conflicts, we solved issue #42 before #33 and #34. 

To fix it, as issues #33 and #34 introducing the feature of building light and dark clouds in the sky and nothing else, you may adjust the code so it make sure no blocks other than light or dark clouds can be attached to any given clouds.

Thank you for your effort.

**Review by Lurk390 — APPROVED** [review]
Looks fine to me and passes the tests

File: tempus_fugit_minecraft/block.py [hunk 1]
```diff
@@ -61,7 +61,15 @@ def tex_coords(top: tuple, bottom: tuple, side: tuple) -> list:
...
-LIGHT_CLOUD = Block("LIGHT_CLOUD", ((3, 0), (3, 0), (3, 0)), is_breakable=False, is_collidable=False, can_build_on=False)
+LIGHT_CLOUD = Block("LIGHT_CLOUD", ((3, 0), (3, 0), (3, 0)), is_breakable=True, is_collidable=False, can_build_on=True)
+"""!
+Issue fogleman#33 and issue#34 were created on June 19 to let players create clouds anywhere and may build their homes on clouds
+issue#42 was created on June 21 that block the user to build on clouds!
+@w1158mxs left a preview on June 22 in issue#42 that will use the key it was mentioned in issue#33 and issue#34 to build clouds and will let players build their homes on clouds!
+while issue#80 used keys
+@w1158mxs uses another key to build clouds
+@see README.md to know how to play it
+"""
```
[the remark this answers is not in the document]
Comment (w1158mxs) [reply tempus_fugit_minecraft/block.py]: I agree!
I removed it now! I'll make it short and direct in oxygen style in another pull request.

**Review by w1158mxs — COMMENTED** [review]
fix [issue#34]

**Review by criticalglitch — COMMENTED** [review]
I'm fairly certain this change is trivial.

**Review by bakursait2 — CHANGES_REQUESTED** [review]
I don't think we need to attach the tests results to the main directory. We need to do it in the PR itself.

Please, remove it from here, and add it as a comment to the PR that is related to this branch!
To do it: 
1. go to your [PR](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/pull/105) 
2. Copy the image for the test result in your computer (`ctrl+c`)
3. add it in the comment by pasting it (`ctrl+v`), and you will get something similar to this:
image
4. Then you are good to go.

Thank you for your effort.

**Comment by w1158mxs** [comment]
> I'm fairly certain this change is trivial.

If you want to make players build clouds only on clouds you can create a new issue

**Review by bakursait2 — APPROVED** [review]

**Review by criticalglitch — CHANGES_REQUESTED** [review]
Please rebase your PR, there are conflicts.

**Comment by w1158mxs** [comment]
Player_Create_Light_Clouds_Test

Tested on local computer up to date

**Pull request closed without merging**